pyproject.toml

```
@@ -26,12 +26,12 @@ botocore = "^1.34.85"
 click = "^8.1.3"
 cloudfoundry-client = "1.35.2"
 mypy-boto3-codebuild = "^1.26.0.post1"
-python = "^3.9"
+python = ">3.9.0,<3.9.1 || >3.9.1,<4.0"
 schema = "0.7.5"
 cfn-flip = "1.3.0"
 aiohttp = "^3.8.4"
 certifi = ">=2023.7.22,<2025.0.0"
-cryptography = ">=41.0.3,<44.0.0"
+cryptography = "44.0.1"
```

Can we allow it to take minor and patch releases?
```suggestion
cryptography = ">=44.0.1, <45"
```

---

yeah, wasn't sure about that as we have both in the toml, but that looks like a sensible suggestion